processor/src/main/java/com/linecorp/decaton/processor/metrics/Metrics.java

```diff
@@ -145,6 +145,18 @@ public void close() {
     }
 
     public class TaskMetrics extends AbstractMetrics {
+        public final Timer tasksDeliveryLatency =
+                meter(() -> Timer.builder("tasks.delivery.latency")
+                                 .description("The latency between the time the task is produced and the time the task is consumed")
```

Comment:
We should mention that it might not represent the actual e2e latency depending on how is `timestampMillis` constructed for the task, particularly when `TaskExtractor` interface is used, user might not set appropriate `timestampMillis` (which is OK if user is aware of that, but want to avoid misleading users by letting them to assume e2e latency is somehow way shorter than actually is).

Reply:
That's indeed true. This metric heavily relies on `timestampMillis`. Based on your review comment, I've added a note to the description of the metrics.

I have a question. Should this also include in the description about the case where an inappropriate `timestampMillis` is set when using TaskExtractor or decaton client's `put`? Or is a comment sufficient?

Reply:
I think the current statement is sufficient.